llarp/dnsc.cpp

```diff
@@ -224,14 +227,19 @@ generic_handle_dnsc_recvfrom(dnsc_answer_request *request,
   for(uint32_t i = 0; i < hdr->qdCount; i++)
   {
     question = decode_question(castBufc, &pos);
-    //llarp::LogDebug("Read a question, now at ", std::to_string(pos));
+    std::unique_ptr< dns_msg_question > unique(question);
+    request->packet.questions.push_back(std::move(unique));
```

Comment:
instead just do:

```c++
request->packet.questions.emplace_back(question);
```

Reply:
Ah thank you! I knew this could be done better.